lib/matplotlib/axes/_axes.py

```diff
@@ -84,9 +84,37 @@ def _plot_args_replacer(args, data):
                          "multiple plotting calls instead.")
 
 
+def _make_inset_locator(rect, trans, parent):
+    """
+    Helper function to locate inset axes, used in
+    `.Axes.inset_axes_from_bounds`.
+
+    A locator gets used in `Axes.set_aspect` to override the default
+    locations...  It is a function that takes an axes object and
+    a renderer and tells `set_aspect` where it is to be placed.
+
+    Here *rect* is a rectangle [l, b, w, h] that specifies the
+    location for the axes in the transform given by *trans* on the
+    *parent*.
+    """
+    _rect = mtransforms.Bbox.from_bounds(*rect)
+    _trans = trans
+    _parent = parent
+
+    def inset_locator(ax, renderer):
+        bbox = _rect
```

Comment:
Is this line actually doing anything other than renaming a variable?  I think it is superfluous.